webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -323,6 +323,10 @@ export class SchemaAdminSettings extends React.PureComponent<Props, State> {
             return setting.dynamic_value(this.state[setting.key], this.props.config, this.state);
         }
 
+        if (this.state[setting.key] === undefined && 'default' in setting) {
+            return setting.default;
+        }
```

Comment:
This way we set all the values to the default if needed, instead of leaving them undefined.